talent/job-descriptions/offers/qa-senior.md

```diff
@@ -0,0 +1,30 @@
+Senior QA Engineer
+------------------
+
+### [Location & Seniority](../location-seniority-section.md)
+
+**Compensation:** 49-53,000 Euros + 0.095% [stock options](https://github.com/src-d/guide/blob/master/talent/esop.md) for senior engineers
+
+### [Company](../company-section.md)
+
+### [Engineering at source{d}](../engineering-section.md)
+
+### Role 
+
+source{d} is building a new QA team to work across the whole software stack. This involves testing command line tools, web applications and distributed systems. Systems under test are written mainly in Go, Python, Scala and JavaScript.
+
+- Strong software engineering skills are required for this position. In particular, we expect testing code to attain the same quality level required for code under testing.
+
+- No previous experience with any particular programming language is required, but the candidate should be willing to learn new languages such as Go and Python.
+
+- Notions of distributed systems are desirable. Kubernetes and Apache Spark are be used intensively.
+
+- The role will involve improving continuous integration at source{d}, currently relying mainly on Travis CI and Appveyor. Experience with Jenkins CI is desirable but not required.
+
+- The role will involve helping to improve QA and release processes as a whole.
```

Comment:
What do you mean by "release processes as a whole"?

Reply:
Simplified the sentence to just "the release process". That is, they way that we create and announce new versions and what are the preconditions for a stable release to be created.